### PR TITLE
Address JIRA SLE-5607

### PR DESCRIPTION
**Pull request by lproven**
### Description
@jankara queried my table for the NVDIMM diagram in the new Troubleshooting §.
So, I have replaced it with an SVG diagram created in LibreOffice Draw. 

This is my first attempt at doing this so I may not have done it right. I took the green colour as a hex value from the SUSE template. Others (black, white, mid grey) are LO defaults.

### Checklist
* Check all items that apply.

*Are backports required?*

- [x] To maintenance/SLE15SP1
- [x] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: images/src/svg/nvdimm-layout.svg [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      <g class="com.sun.star.drawing.TextShape">
+       <g id="id10">
+        <rect class="BoundingBox" stroke="none" fill="none" x="1027" y="4801" width="4441" height="1774"/>
+        <text class="TextShape"><tspan class="TextParagraph" font-family="Liberation Sans, sans-serif" font-size="635px" font-weight="400"><tspan class="TextPosition" x="1277" y="5502"><tspan fill="rgb(0,0,0)" stroke="none">NVDIMM 1</tspan></tspan></tspan></text>
+       </g>
+      </g>
+      <g class="com.sun.star.drawing.TextShape">
+       <g id="id11">
+        <rect class="BoundingBox" stroke="none" fill="none" x="1000" y="8116" width="4441" height="1775"/>
+        <text class="TextShape"><tspan class="TextParagraph" font-family="Liberation Sans, sans-serif" font-size="635px" font-weight="400"><tspan class="TextPosition" x="1250" y="8817"><tspan fill="rgb(0,0,0)" stroke="none">NVDIMM 1</tspan></tspan></tspan></text>
```
[the remark this answers is not in the document]
Comment (lproven) [reply images/src/svg/nvdimm-layout.svg]: Just FYI, I already showed him an earlier version of the diagram and he liked it.

[the remark this answers is not in the document]
Comment (lproven) [reply images/src/svg/nvdimm-layout.svg]: Suggested revisions to diagram incorporated and approved

File: xml/hardware_nvdimm.xml [hunk 2]
```diff
@@ -825,6 +825,216 @@ I/O size (minimum/optimal): 4096 bytes / 4096 bytes</screen>
...
+   and regions use it, then the data in other, unaffected namespaces can be
+   backed up, the server shut down and the NVDIMM replaced. 
+  </para>
+  <sect2>
+   <title>Locating a Failed Module</title>
+   <para>
+    A set of NVDIMMs are located in DIMM slots on the motherboard of the server.
+   </para>
+   <para>
+    In the resultant space, the operating system creates one or more namespaces,
```
[the remark this answers is not in the document]
Comment (lproven) [reply xml/hardware_nvdimm.xml]: Sentences revised

**Comment by taroth21** [comment]
According to Liam, Jan gave a 'good to go' from the technical side, so squashed and merged now.

**Pull request merged**